Cargo.toml

```diff
@@ -10,7 +10,7 @@ members = [ "tools", "ffi", "backend", "widgets" ]
 
 [dependencies]
 cascade = "1"
-cairo-rs = "0.14.0"
+cairo-rs = "0.15.0"
```

Comment:
Good catch. But actually this dependency isn't needed now since the code using cairo imports it as `gtk::cairo`.

Reply:
Oh neat. I dropped it completely then. :+1: